config/config_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config_test
+
+import (
+	"testing"
+
+	"github.com/ligato/cn-infra/config"
+	"github.com/onsi/gomega"
```

Comment:
Use dot import for more readable tests. 
`Expect(pluginConfig).ShouldNot(BeNil())` vs. `gomega.Expect(pluginConfig).ShouldNot(gomega.BeNil())`

Reply:
Done